### PR TITLE
[PWA] Add vitest; add team sponsor parsing & accordion

### PR DESCRIPTION
**Pull request by tervay**
image
image

----

image
image

---

image
image

---

image
image

---

image
image

---

image

---
[//]: # (BEGIN SAPLING FOOTER)
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/the-blue-alliance/the-blue-alliance/pull/6303).
* #6305
* __->__ #6303

### CODE REVIEW

**Comment by tervay** [comment]
Changed to font-normal and outline variant

image

File: pwa/app/routes/team.$teamNumber.tsx [hunk 1]
```diff
@@ -105,10 +115,29 @@ export default function TeamPage(): JSX.Element {
...
-        <InlineIcon displayStyle={'flexless'}>
-          <BiInfoCircleFill />
-          {team.name}
-        </InlineIcon>
+        <Accordion type="single" collapsible>
+          <AccordionItem value="item-1" className="border-0">
+            <AccordionTrigger className="p-0 text-left font-normal">
+              <InlineIcon displayStyle={'flexless'}>
+                <BiInfoCircleFill />
+                {team.school_name ?? 'Family/Community'}
```
Comment (fangeugene) [inline pwa/app/routes/team.$teamNumber.tsx:123]: I'm not sure this is something we can rely on, especially for older teams that went defunct before `school_name` was provided (e.g. 1717).

File: pwa/app/routes/team.$teamNumber.tsx [hunk 2]
```diff
@@ -105,10 +115,29 @@ export default function TeamPage(): JSX.Element {
           {team.city}, {team.state_prov}, {team.country}
         </InlineIcon>
 
-        <InlineIcon displayStyle={'flexless'}>
-          <BiInfoCircleFill />
-          {team.name}
-        </InlineIcon>
+        <Accordion type="single" collapsible>
```
Comment (fangeugene) [inline pwa/app/routes/team.$teamNumber.tsx:118]: Should we not render the accordion if there are zero sponsors?

**Comment by tervay** [comment]
image

image

---

removed accordion on 0 sponsors:
image

**Pull request merged**